### PR TITLE
Lemontrees123 want to join the Istio org

### PR DESCRIPTION
**Pull request by lemontrees123**
## I want to join the Istio org!

- [x] I have reviewed the [community membership guidelines](https://github.com/istio/community/blob/master/ROLES.md#member)
- [x] I have reviewed the [contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md)
- [x] I have enabled [2FA on my GitHub account](https://github.com/settings/security)
- [x] I have joined the [Istio discussion board](https://discuss.istio.io) and subscribed to the [Contributors](https://discuss.istio.io/c/contributors) category
- [x] I have joined [Istio's Slack workspace](https://slack.istio.io)
- [x] I have written my company name or "Individual", if not affliated with a company, here: ___DaoCloud_______
- [x] I have provided a link to at least one PR that I have successfully pushed to one of the Istio repos, here: _https://github.com/istio/istio.io/pull/13538_________

### CODE REVIEW

**Comment by craigbox** [comment]
Hi @lemontrees123,

Please fill in the template, including your employer and PRs that you've had merged. 
You will also need to sort the list to put your ID in alphabetical order.

(It looks like you're on the docs team at DaoCloud?)

**Comment by lemontrees123** [comment]
> Hi @lemontrees123,
> 
> Please fill in the template, including your employer and PRs that you've had merged. You will also need to sort the list to put your ID in alphabetical order.
> 
> (It looks like you're on the docs team at DaoCloud?)

**Comment by lemontrees123** [comment]
Thanks for the tip
I have reviewed the [community membership guidelines](https://github.com/istio/community/blob/master/ROLES.md#member)
 I have reviewed the [contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md)
 I have enabled [2FA on my GitHub account](https://github.com/settings/security)
 I have joined the [Istio discussion board](https://discuss.istio.io/) and subscribed to the [Contributors](https://discuss.istio.io/c/contributors) category
 I have joined [Istio's Slack workspace](https://slack.istio.io/)
 I have written my company name or "Individual", if not affliated with a company, here: ___Daocloud_______
 I have provided a link to at least one PR that I have successfully pushed to one of the Istio repos, here: __https://github.com/istio/istio.io/compare/master...lemontrees123:istio.io:lemon01________

**Comment by craigbox** [comment]
(Thanks; for ease of future reference please do this by editing the issue description and ticking the boxes.)

File: .DS_Store
Comment (craigbox) [inline .DS_Store:1]: remove this file please

Comment (lemontrees123) [reply .DS_Store:1]: I can't delete this file because the deletion operation is grey

Comment (lemontrees123) [reply .DS_Store:1]: /
WX20230727-174635

Comment (lemontrees123) [reply .DS_Store:1]: I already took care of it

**Review by kfaseela — APPROVED** [review]
Welcome :)

**Pull request merged**